doc/api/worker_threads.md

```diff
@@ -4,8 +4,8 @@
 
 > Stability: 1 - Experimental
 
-The `worker_threads` module enables the use of threads with message channels
-between them. To access it:
+The `worker_threads` module enables the use of threads that execute JS code
+in parallel. To access it:
```

Comment:
Maybe something more like "that allow parallel execution contexts for JS code"?

Reply:
@Fishrock123 I’m not sure … “execution context” sounds like a very generic thing?